src/js/components/Button/StyledButton.js

```diff
@@ -128,6 +128,7 @@ const plainStyle = css`
   border: none;
   padding: 0;
   text-align: inherit;
+  min-height: min-content;
```

Comment:
`min-content` doesn't seem to be generally supported: https://developer.mozilla.org/en-US/docs/Web/CSS/min-height